ray-operator/hack/config.yaml

```diff
@@ -0,0 +1,10 @@
+processor:
+  ignoreTypes:
```

Comment:
Would you mind explaining this file? Thanks!

Reply:
It's a [configuration](https://github.com/elastic/crd-ref-docs#configuration) provided by `crd-ref-docs` to define regex for excluding some types or type fields from the generated documentation. 

In the current config, I ignore the types with `List` and `Status` suffix. I am not sure whether to show them in the documentation. What do you think?

Reply:
Thank you for the explaination!  Ignoring the types with List and Status suffix makes sense to me.